watcher.yml

```diff
@@ -0,0 +1,7 @@
+artifacts:
+  - url: https://raw.githubusercontent.com/safing/intel-data/master/spn/main-intel.json
+    scope: intel/spn
```

Comment:
Not sure how the IntelHub handles this, but the full identifier is "all/intel/...".

Reply:
there are additional configuration options for each artifact for `architecture` and `os`. The path `all` or `windows_amd64` is built from those values.